src/plugins/explore/public/components/visualizations/gauge/gauge_vis_options.tsx

```diff
@@ -0,0 +1,165 @@
+import { StyleAccordion } from '../style_panel/style_accordion';
+import { AxesSelectPanel } from '../style_panel/axes/axes_selector';
+import { DebouncedText, DebouncedTruncateGaugeBaseField } from '../style_panel/utils';
+import { ValueCalculationSelector } from '../style_panel/value/value_calculation_selector';
+
+export type GaugeVisStyleControlsProps = StyleControlsProps<GaugeChartStyleControls>;
+
+export const GaugeVisStyleControls: React.FC<GaugeVisStyleControlsProps> = ({
+  styleOptions,
+  onStyleChange,
+  numericalColumns = [],
+  categoricalColumns = [],
+  dateColumns = [],
+  availableChartTypes = [],
+  selectedChartType,
+  axisColumnMappings,
+  updateVisualization,
+}) => {
+  const updateStyleOption = <K extends keyof GaugeChartStyleControls>(
+    key: K,
+    value: GaugeChartStyleControls[K]
+  ) => {
+    onStyleChange({ [key]: value });
+  };
+
+  // The mapping object will be an empty object if no fields are selected on the axes selector. No
+  // visualization is generated in this case so we shouldn't display style option panels.
+  const hasMappingSelected = !isEmpty(axisColumnMappings);
+
+  return (
+    <EuiFlexGroup direction="column" gutterSize="none">
+      <EuiFlexItem grow={false}>
+        <AxesSelectPanel
+          numericalColumns={numericalColumns}
+          categoricalColumns={categoricalColumns}
+          dateColumns={dateColumns}
+          currentMapping={axisColumnMappings}
+          updateVisualization={updateVisualization}
+          chartType="gauge"
+        />
+      </EuiFlexItem>
+      {hasMappingSelected && (
+        <>
+          <EuiFlexItem>
+            <StyleAccordion
+              id="gaugeValueOptions"
+              accordionLabel={i18n.translate('explore.stylePanel.gaugeValueOptions', {
+                defaultMessage: 'Value options',
+              })}
+              initialIsOpen={false}
+            >
+              <EuiFormRow
+                label={i18n.translate('explore.vis.gauge.calculation', {
+                  defaultMessage: 'Calculation',
+                })}
+              >
+                <ValueCalculationSelector
+                  selectedValue={styleOptions.valueCalculation}
+                  onChange={(value) => updateStyleOption('valueCalculation', value)}
+                />
+              </EuiFormRow>
+            </StyleAccordion>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <StyleAccordion
+              id="gaugeSection"
+              accordionLabel={i18n.translate('explore.stylePanel.gauge', {
+                defaultMessage: 'Gauge',
+              })}
+              initialIsOpen={true}
+            >
+              <EuiSplitPanel.Inner paddingSize="s" color="subdued">
+                <EuiText size="s" style={{ fontWeight: 600 }}>
+                  {i18n.translate('explore.stylePanel.gauge.threshold.splitPanel', {
+                    defaultMessage: 'Threshold',
```

Comment:
Can we move Threshold to have it's own `StyleAccordion`? So we have a dedicated panel for threshold settings, this will align the experience among different charts